tasks/run.js

```diff
@@ -196,6 +196,7 @@ module.exports = function (program, utils) {
 		.option('--key <file>', 'Specify a certificate key to use with HTTPS. Use with --https.')
 		.option('--local-apps <apps>', 'Specify extra apps that are running locally, as comma-seperated `[name]=[port]`, e.g. `service-worker=3001,front-page=3002`')
 		.option('-p --port <port>', 'Port to run the router through', 5050)
+		.option('--router <router>', 'Router to run using', 'next-router')
```

Comment:
or should this be just `next`?

Pro - more intuitive to type: `--router next` or `--router kat`
Con - less flexible - can't have a router that doesn't end in `-router`... but then again, why would you?
Con - a bit more magic